src/confluent_kafka/src/Consumer.c

```diff
@@ -1341,10 +1458,16 @@ static void Consumer_rebalance_cb (rd_kafka_t *rk, rd_kafka_resp_err_t err,
 			return;
 		}
 
-		result = PyObject_CallObject(
-			err == RD_KAFKA_RESP_ERR__ASSIGN_PARTITIONS ?
-			self->u.Consumer.on_assign :
-			self->u.Consumer.on_revoke, args);
+                if (err == RD_KAFKA_RESP_ERR__ASSIGN_PARTITIONS)
+                        cb = self->u.Consumer.on_assign;
+                else {
+                        if (rd_kafka_assignment_lost(rk) &&
+                            self->u.Consumer.on_lost)
+                                cb = self->u.Consumer.on_lost;
+                        else
+                                cb = self->u.Consumer.on_revoke;
+                }
+		result = PyObject_CallObject(cb, args);
```

Comment:
nit: reinsert the empty line for readability.

Reply:
? 

what would improve readability is coding convention that specifies {} must always used (as is enforced in all new language these days).